docker/1.4.1/py2/mms-entrypoint.py

```diff
@@ -0,0 +1,13 @@
+import shlex
```

Comment:
Add copyright on the top of the file

Reply:
I don't see a sample for a copyright in the master branch.